### DISCUSSION TITLE
Can we add background image on search box ?

### DISCUSSION THREAD

**Discussion by spo365** [microsoft-search/pnp-modern-search#1628] [Ideas]
**Is your feature request related to a problem? Please describe.**
Please provide option add background image in search box web-part 

**Describe the solution you'd like**
It should similar option like we have when we are adding a new pages layout.

**Additional context**
![image](https://user-images.githubusercontent.com/5062256/79628107-76ac7780-8181-11ea-94fe-e2cd12dc955b.png)

**Comment by wobba** [comment]
Not today, and how do you envision that should look? Around or inside the search field?

Unsure what the pasted image illustrates.

**Comment by spo365** [comment]
Some option where i can pass the background image on my sharepoint search box. 
If you have any suggestion i can try to change.
![image](https://user-images.githubusercontent.com/5062256/79691170-55db4380-82a1-11ea-8541-a3e3aa6769a9.png)

**Comment by wobba** [comment]
Ok, so add an image full width behind in the web part area and transparency settings for the search box..is that correct?

**Comment by spo365** [comment]
Yes.
